src/ipa_pointcloud_to_laserscan_nodelet.cpp

```diff
@@ -149,7 +149,7 @@ void IpaPointCloudToLaserScanNodelet::cloudCb(const sensor_msgs::PointCloud2Cons
     }
     catch (tf2::TransformException ex)
     {
-      NODELET_WARN_STREAM("Transform failure: " << ex.what());
+      NODELET_DEBUG_STREAM("Transform failure: " << ex.what());
```

Comment:
Just a side note: This is `tf2`, so there is no `waitForTransform`. The fourth parameter in `lookupTransform` would need to be adjusted accordingly (http://wiki.ros.org/tf2/Tutorials/tf2%20and%20time%20(C%2B%2B)#Wait_for_transforms).